mlflow/store/_unity_catalog/registry_oss/rest_store_oss.py

```diff
@@ -58,28 +77,76 @@ def create_registered_model(self, name, tags=None, description=None):
                 catalog_name=catalog_name,
                 schema_name=schema_name,
                 comment=comment,
-                tags=tags,
             )
         )
         registered_model_info = self._call_endpoint(CreateRegisteredModel, req_body)
         return registered_model_from_uc_oss_proto(registered_model_info)
 
     def update_registered_model(self, name, description):
-        raise NotImplementedError("Method not implemented")
+        full_name = get_full_name_from_sc(name, None)
+        [catalog_name, schema_name, model_name] = full_name.split(".")
+        comment = description if description else ""
+        req_body = message_to_json(
+            UpdateRegisteredModel(
+                full_name=full_name,
+                new_name=model_name,
```

Comment:
`new_name` should be left blank if it's just the same as `full_name`. in mlflow there's a separate method called `rename_regsitered_model` that achieves the renaming functionality by leveraging the `UpdateRegisteredModel` RPC.